cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py

```diff
@@ -5,10 +5,13 @@
 
 import boto3
 
+from cloud_governance.common.logger.init_logger import logger
+
 
 class CloudTrailOperations:
     SEARCH_SECONDS = 10
     SLEEP_SECONDS = 120
+    DAYS_GO_BACK = 30
```

Comment:
What it means?

Reply:
If I didn't find a username in the tags, I search in the cloud trail. Some resources don't have create time. So I search for the last 30 days

Reply:
Why 30 and not 90 days ?

Reply:
I thought 30 days is large enough.